client/network/src/service.rs

```diff
@@ -812,6 +885,13 @@ impl<B, H> NetworkStateInfo for NetworkService<B, H>
 	}
 }
 
+/// Error returned by [`NetworkService::send_notification`].
+#[derive(Debug, derive_more::Display, derive_more::Error)]
+pub enum SendNotificationError {
```

Comment:
```suggestion
pub enum SendNotificationsError {
```

Would be easier to match to the corresponding function `send_notificationS`.